### PR TITLE
Feature/server dockerize, small cleanup

### PR DESCRIPTION
**Pull request by fractalawareness**
Hello :)
I've just started playing with SignificantTrades and would like to share dockerized version of the server part.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,4 +92,4 @@ All options can be set using CLI
 - Setting port & pair and restrict to some exchanges `node index port=3002 pair=ETHUSD bitmex bitfinex kraken`
 
 *Like whats been done here ?* Donate<br>
-[3NuLQsrphzgKxTBU3Vunj87XADPvZqZ7gc](bitcoin:3NuLQsrphzgKxTBU3Vunj87XADPvZqZ7gc)
\ No newline at end of file
+[3NuLQsrphzgKxTBU3Vunj87XADPvZqZ7gc](bitcoin:3NuLQsrphzgKxTBU3Vunj87XADPvZqZ7gc)
```
Comment (fractalawareness) [inline README.md]: Haven't changed anything here. It's code editor replaced spaces/tabs.

**Comment by fractalawareness** [comment]
Any suggestions for improvements in this PR?

File: src/storage/files.js [hunk 2]
```diff
@@ -51,7 +52,7 @@ class FilesStorage {
       filename += `-${('0' + date.getSeconds()).slice(-2)}`
     }
 
-    return filename.replace(/\s+/g, '')
+    return path.join(this.options.filesLocation, filename.replace(/\s+/g, ''));
```
Comment (Tucsky) [inline src/storage/files.js:55]: this.options.filesLocation is concatened twice (here and line 37)

Comment (fractalawareness) [reply src/storage/files.js]: Thanx, fixed 👍

**Comment by fractalawareness** [comment]
Thank you for approve. I have no write access to this repo, so will you merge it please?

**Pull request merged**